src/App.js

```diff
@@ -0,0 +1,34 @@
+import './App.css';
```

Comment:
- [ ] It would be better if our component structure includes the following: `src/redux/configureStore.js` file so that we can set up our redux store to connect with the react app

Reply:
Well solved